extra/mariabackup/xtrabackup.cc

```diff
@@ -1267,23 +1267,19 @@ struct my_option xb_client_options[]= {
      GET_STR, REQUIRED_ARG, 0, 0, 0, 0, 0, 0},
 
     {"compress", OPT_XTRA_COMPRESS,
-     "Compress individual backup files using the "
-     "specified compression algorithm. Currently the only supported algorithm "
-     "is 'quicklz'. It is also the default algorithm, i.e. the one used when "
-     "--compress is used without an argument.",
+     "This option was deprecated starting with MariaDB 10.1.31 and 10.2.13 "
```

Comment:
Where did you get these versions from?

If there's a reason, explain it in the commit message, otherwise a non-versioned deprecation notice is ok.

Reply:
this was in mariabackup knowledgebase [documentation](https://mariadb.com/kb/en/mariabackup-options/#-compress), I took it from there.